### PR TITLE
Add configuration documents for the Event Bus component

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Created configuration docs for these Event Bus charts:
   -  [`event-bus`](https://github.com/kyma-project/kyma/tree/master/resources/event-bus)
   - [`nats-streaming`](https://github.com/kyma-project/kyma/tree/master/resources/nats-streaming)
- Fixed dead links in the main `README.md` document for the `docs` folder

**Related issue(s)**
See also #4070

### CODE REVIEW

File: docs/event-bus/05-01-event-bus.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.publishKnative.maxRequests** | Specifies the maximum number of parallel Event requests that **publish-knative** can process. If you raise this value, you may also have to increase memory resources for the Event Bus to handle the higher load. | `16` |
+| **global.publishKnative.maxRequestSize** | Specifies the maximum size of one Event. If you raise this value, you may also have to increase memory resources for the Event Bus to handle the higher load. | `65536` |
+| **global.publishKnative.resources.limits.memory** | Specifies memory limits set for **publishKnative**. | `32M` |
```
Comment (mmitoraj) [inline docs/event-bus/05-01-event-bus.md]: **publishKnative** or **publish-knative**?

File: docs/event-bus/05-01-event-bus.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+| **global.subscriptionControllerKnative.resources.limits.memory** | Specifies memory limits set for **subscription-controller-knative**. | `32M` |
```
Comment (mmitoraj) [inline docs/event-bus/05-01-event-bus.md]: **subscription-controller-knative** or **subscriptionControllerKnative**?

File: docs/event-bus/05-02-nats-streaming.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.natsStreaming.persistence.maxAge** | Specifies the time for which the given Event is stored in NATS Streaming. | `24h` |
+| **global.natsStreaming.persistence.size** | Specifies the size of the persistence volume in NATS Streaming. | `1Gi` |
+| **global.natsStreaming.resources.limits.memory** | Specifies the memory limits for NATS Streaming. | `256M` |
+| **global.natsStreaming.channel.maxInactivity** | Specifies the time after which the autocleaner removes from the NATS Streaming database all backing resources related to a given Event type if there is no activity for this Event type. | `48h` |
```
Comment (mmitoraj) [inline docs/event-bus/05-02-nats-streaming.md]:
```suggestion
| **global.natsStreaming.channel.maxInactivity** | Specifies the time after which the autocleaner removes all backing resources related to a given Event type from the NATS Streaming database if there is no activity for this Event type. | `48h` |
```

Comment (mmitoraj) [reply docs/event-bus/05-02-nats-streaming.md]: it seems more natural for me, but do as you please ;)

Comment (kazydek) [reply docs/event-bus/05-02-nats-streaming.md]: Yes, that was a natural path for me as well :) (what + where) but the length of the "what" somehow made me change my mind ;) but you are right, it needs reverting.

File: docs/event-bus/05-01-event-bus.md (hunk 1 above)
Comment (kazydek) [reply docs/event-bus/05-01-event-bus.md]: Hmm, I kinda used the naming/formatting approach from the Event Bus [Architecture](https://kyma-project.io/docs/1.1/components/event-bus/#architecture-architecture) doc.

File: docs/event-bus/05-01-event-bus.md (hunk 2 above)
Comment (kazydek) [reply docs/event-bus/05-01-event-bus.md]: Same case as the one above.

File: docs/event-bus/05-01-event-bus.md (hunk 1 above)
Comment (kazydek) [reply docs/event-bus/05-01-event-bus.md]: I changed it to adjust to the parameter names.

File: docs/event-bus/05-01-event-bus.md (hunk 2 above)
Comment (kazydek) [reply docs/event-bus/05-01-event-bus.md]: I changed it to adjust to the parameter names.

**Pull request merged**